src/status_im/contexts/settings/wallet/events.cljs

```diff
@@ -221,3 +221,29 @@
 
 (rf/reg-event-fx :wallet/import-missing-keypair-by-private-key-failed
  import-missing-keypair-by-private-key-failed)
+
+(defn make-partially-operable-accounts-fully-operable-success
+  [{:keys [db]} [addresses]]
+  (let [key-uids-to-update (data-store/map-addresses-to-key-uids db addresses)
+        key-uids-set       (set key-uids-to-update)]
```

Comment:
Since the method `map-addresses-to-key-uids` returns a set, `key-uids-set` is redundant here.